ruuvitag_sensor/ruuvi.py

```diff
@@ -157,7 +157,7 @@ def _get_ruuvitag_datas(macs=[], search_duratio_sec=None, run_flag=RunFlag(), bt
                 data_iter.send(StopIteration)
                 break
             # Check MAC whitelist if advertised MAC available
-            if ble_data[0] and macs and not ble_data[0] in macs:
+            if ble_data[0] and macs and not ble_data[0].upper() in map(str.upper, macs):
```

Comment:
We could have a new list before for loop, so new map is not created each time new data is found?
```py
allowed_macs = map(str.upper, macs)
...
if ble_data[0] and allowed_macs and not ble_data[0].upper() in allowed_macs:
```